### DISCUSSION TITLE
The Pressure Stabilizer MUST TO inject automatically if it is inserted into a Auto-Injector Headset. APS in the autoinjector will open up the prospect of wearing variety of different beautiful costumes.

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#13239] [Ideas and suggestions]
Currently, pressure kills instantly if you are under pressure for more than 5 seconds.
In my humble opinion, the mechanism of pressure action needs to be reconsidered. Pressure does not kill instantly, so the health level of a person subjected to pressure should decrease, although quickly, but gradually. In this case, the APS inserted into the auto-injector MUST to be applied.
Those parts of the body that do not contain gas are not compressible and do not deteriorate due to pressure.
The pressure compresses the air in the lungs, preventing a person from breathing even if he is wearing an oxygen mask, and the person dies from asphyxia. Thus, the man dies from lack of oxygen. And this happens within 5 minutes, not 5 seconds.
One of the most common reasons for the death of bots (and people) is that they walk around without a diving suit and do not have time to reach it if there is a large hole. Because of this, I always wear a diving suit. If only APS in the autoinjector had worked, it would have been possible to walk without a diving suit but with APS in the autoinjector.
To control auto-inject APS not needed AI, if the stabilizer is wasted it’s not so bad, but APS will save our life in case of real pressure danger.
Without redesign mechanism of pressure, just make the APS in the auto-injector work if the pressure is life-threatening and lasts more than 3 seconds.
It’s not difficult but very useful.
Now in the second half of the game we all have to wear a diving suits all the time (except symbiont husks of course).
APS in the autoinjector will open up the prospect of wearing variety of different beautiful costumes.

**Comment by G-M-twostay** [comment]
The idea of a functional pressure stabilizer in autoinjector headset sounds great.

**Comment by apollodeathstar** [comment]
I hope they do this

**Comment by SomeRandomNoobKekeke** [comment]
someone did that https://steamcommunity.com/sharedfiles/filedetails/?id=3637826874
